typegate/src/runtimes/wasm.ts

```diff
@@ -61,33 +62,44 @@ export class WasmRuntime extends Runtime {
       return [stage.withResolver(() => ({}))];
     }
 
-    return [stage.withResolver(({ _: { parent } }) => {
-      if (stage.props.parent == null) { // namespace
-        return {};
-      }
-      const resolver = parent[stage.props.node];
-      return typeof resolver === "function" ? resolver() : resolver;
-    })];
+    const _sameRuntime = Runtime.collectRelativeStages(stage, _waitlist);
+
```

Comment:
@destifo This seems to be the reason why it was was failing.
If intentionnal (eg. refactor), you can  refer to how it is done in other runtimes.

Reply:
Yeah, you are right! how did it affect the result?

Reply:
This function call is context sensitive. It will mutate the waitlist.

Reply:
OK, I see. Thanks! The output log was the same and I was thinking the error was happening elsewhere.